File: emailSender/src/main/java/com/przemo/emailsender/models/Attachment.java
package com.przemo.emailsender.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@ToString
@NoArgsConstructor
@Getter
@Setter
public class Attachment {

    private String fileName;
    private String attachment_encoded_b64;
    private byte[] attachment_decoded_b64;

}

File: rentCarAngular2/src/app/attachment.ts
export class Attachment {
    fileName:string;
    attachment_encoded_b64:string;

    constructor(fileName:string,attachment_encoded_b64:string){
        this.attachment_encoded_b64 = attachment_encoded_b64;
        this.fileName = fileName;
    }

    setFielName(fileName){
        this.fileName = fileName;
    }

    setAttachmentEncodedB64(attachment_encoded_b64){
        this.attachment_encoded_b64 = attachment_encoded_b64;
    }

}
